lib/chef/knife/ssl_fetch.rb

```diff
@@ -28,6 +28,8 @@ class SslFetch < Chef::Knife
         require 'socket'
         require 'uri'
         require 'openssl'
+        require_relative 'proxified_socket'
```

Comment:
No it doesn't, in fact it's generally considered good to use because it is much much faster than `require` since it doesn't process the load paths.